DOCS/man/console.rst

```diff
@@ -14,50 +14,82 @@ Keybindings
 ESC
     Hide the console.
 
-ENTER
+ENTER, Ctrl+J and Ctrl+M
     Run the typed command.
 
 Shift+ENTER
     Type a literal newline character.
 
-Ctrl+LEFT and Ctrl+RIGHT
-    Move cursor to previous/next word.
+Ctrl+LEFT and Alt+B
+    Move the cursor to the beginning of the current word, or if between words,
+    to the beginning of the previous word.
 
-UP and DOWN
-    Navigate command history.
+Ctrl+RIGHT and Alt+F
+    Move the cursor to the end of the current word, or if between words, to the
+    end of the next word.
+
+Ctrl+BACKSPACE and Ctrl+W
+    Delete text from the cursor to the beginning of the current word, or if
+    between words, to the beginning of the previous word.
+
+UP and Ctrl+P
+    Move back in the command history.
+
+DOWN and Ctrl+N
+    Move forward in the command history.
 
 PGUP
     Go to the first command in the history.
 
 PGDN
-    Stop navigating command history.
+    Stop navigating the command history.
 
 INSERT
     Toggle insert mode.
 
+Ctrl+V
+    Paste text (uses the clipboard on X11 and Wayland).
+
 Shift+INSERT
     Paste text (uses the primary selection on X11 and Wayland).
 
-TAB
+TAB and Ctrl+I
     Complete the command or property name at the cursor.
 
-Ctrl+C
-    Clear current line.
-
-Ctrl+K
-    Delete text from the cursor to the end of the line.
-
 Ctrl+L
     Clear all log messages from the console.
 
+Ctrl+A
+    Move the cursor to the start of the current line.
+
+Ctrl+E
+    Move the cursor to the end of the current line.
+
+Ctrl+B
+    Move the cursor to the previous character.
+
+Ctrl+F
+    Move the cursor to the next character.
+
+Ctrl+H
+    Delete the previous character.
+
+Ctrl+D
+    Hide the console if all lines are empty, otherwise delete the next
+    character.
+
+Alt+D
+    Delete text from the cursor to the end of the current word, or if between
+    words, to the end of next word.
+
 Ctrl+U
     Delete text from the cursor to the beginning of the line.
 
-Ctrl+V
-    Paste text (uses the clipboard on X11 and Wayland).
+Ctrl+K
+    Delete text from the cursor to the end of the line.
 
-Ctrl+W
-    Delete text from the cursor to the beginning of the current word.
+Ctrl+C
+    Clear all lines.
```

Comment:
I think we should use consistent terminology here. Some of the docs, like for Ctrl+K and Ctrl+U mention the end and beginning of the "line," but they really clear to the end and beginning of the command, which is "all lines." "All lines" sounds to me like it clears all history lines, though.

I'm not sure what these should say. I realise now that in readline, Ctrl+K and Ctrl+U really do just operate on one line of a multi-line command. Multi-line commands were kind of an afterthought in the mpv console, so I didn't think of making them work that way. Maybe we should just always call it the "current line" or the "current command." As long as it's consistent with Ctrl+K and others.

Reply:
current command isn't right either since you can run `command 1; command 2`. I believe buffer is the correct term; that's what zsh calls it in its documentation.